### DISCUSSION TITLE
X13 AMD Fusion shortcut

### DISCUSSION THREAD

**Discussion by tonythetable** [seerge/g-helper#1354] [Ideas]
Hi, I mainly wanted to say that I'm super impressed by Ghelper and the work that has been done :)

As an idea I wanted to offer one feature which would be for a quick way to disable AMD Fusion Hub, which causes the Flow X13 to auto rotate if it's in portrait modus or is used as a tablet (disabling the keyboad and such) right now I have a workaround for it with two bat files containing the following code

"pnputil /enable-device "PCI\VEN_1022&DEV_15E4&SUBSYS_164A1022&REV_00\4&31F4EC98&0&0741""(enable) and
"pnputil /disable-device "PCI\VEN_1022&DEV_15E4&SUBSYS_164A1022&REV_00\4&31F4EC98&0&0741"" for disable. I wondered if this could be made into a shortcut, potentially with the already Custom feature, but i'm not sure how to do so!

Thanks anyhow!

**Comment by seerge** [comment]
@tonythetable hello, you can just put this commands into .bat file and bind via "Custom" action to any of the M1-M4 keys under Extra settings :)
